packages/wrangler/src/dialogs.ts

```diff
@@ -130,3 +130,28 @@ export async function select<Values extends string>(
 	});
 	return value;
 }
+
+export async function multiselect<Values extends string>(
+	text: string,
+	options: SelectOptions<Values>
+): Promise<Values[]> {
+	if (isNonInteractiveOrCI()) {
+		throw new NoDefaultValueProvided();
+	}
```

Comment:
Nit: It would be nice if we can make it fallback to a defaultValue similar to the other prompts so it could be reused easily.